package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "pytest-html",
+  "version": "1.10.0",
```

Comment:
I think I've generated this file by some automatic command line interface, I'll check which of those are really necessary.